### DISCUSSION TITLE
Ninja Installer & Anti-virus

### DISCUSSION THREAD

**Discussion by enthusiast01** [szapp/Ninja#28] [General]
Ninja installer still triggers anti-virus. I do not use installers for mods because of unknown things that can be put into installers. I only use game mods with installers if they are well known by a large community.

Ninja manual install is simple (2 files). So why not just make a manual option to keep people from worrying about virus?

**Comment by szapp** [comment]
You are assuming the installation is just a file extraction. The setup includes additional actions, i.e. environment checks and adjustments to the installation accordingly.
